components/org.apache.stratos.autoscaler/src/main/java/org/apache/stratos/autoscaler/applications/parser/DefaultApplicationParser.java

```diff
@@ -378,13 +374,13 @@ private Application buildCompositeAppStructure(ApplicationContext applicationCon
 
             // get hostname and cluster id
             ClusterInformation clusterInfo;
-            if (cartridgeInfo.getMultiTenant()) {
+            if (cartridge.getMultiTenant()) {
```

Comment:
This line would have made much sense if the getter was renamed to something like "isMultiTenant" or something, I believe.